aten/src/ATen/native/Convolution.cpp

```diff
@@ -1212,7 +1212,7 @@ std::tuple<Tensor,Tensor,Tensor> _convolution_double_backward( const c10::option
     }
   }
 
-  // Compute gI = convT(ggW, gO.t()) if !transposed
+  // Compute gI = convT(gO, ggW) if !transposed
```

Comment:
nit: re-align the "if"?
Also do we want to mention here where it comes from?

Reply:
Tbh, I don't know where it's coming from, it's your code :-), I also don't exactly know what `.t()` meant in the original comment for a 4d tensor.